### PR TITLE
Niloofar / Weekend trading to ts

### PR DESCRIPTION
**Pull request by niloo-fs**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/landing/weekend-trading/components/_hero.tsx [hunk 1]
```diff
@@ -114,14 +114,35 @@ const TryButton = styled(LinkButton)`
...
+                    <StyledHeader type="hero">
+                        {localize('Ride the trends even on weekends')}
+                    </StyledHeader>
                     <HeroContent direction="column" justify="flex-start">
-                        <Header type="subtitle-1">{content}</Header>
+                        <Header type="subtitle-1">
+                            {
+                                <Localize translate_text="Trade even when most financial markets are closed" />
+                            }
+                        </Header>
```
Comment (sean-binary) [inline src/pages/landing/weekend-trading/components/_hero.tsx:145]: Don't forget to add the semantics for the Headers.
```suggestion
                        <Header as="h2" type="subtitle-1">
                            {
                                <Localize translate_text="Trade even when most financial markets are closed" />
                            }
                        </Header>
```

File: src/pages/landing/weekend-trading/components/_image-text-switching.tsx [hunk 2]
```diff
@@ -86,7 +93,21 @@ const query = graphql`
...
+type ImageTextSwitchingProps = {
+    P2P: {
+        title: React.ReactElement
+        subtitle1: React.ReactElement
+        subtitle2: React.ReactElement
+        subtitle_mobile1: React.ReactElement
+        subtitle_mobile2: React.ReactElement
+        image_name: string
+        image_alt: string
+    }[]
```
Comment (sean-binary) [inline src/pages/landing/weekend-trading/components/_image-text-switching.tsx:106]: Please make this a `P2PType` and pass in a `P2PType[]` instead.

File: src/pages/landing/weekend-trading/components/_image-text-switching.tsx [hunk 3]
```diff
@@ -127,15 +144,6 @@ const ImageTextSwitching = ({ P2P, reverse, two_title }) => {
                                 <Show.Mobile>
```
Comment (sean-binary) [inline src/pages/landing/weekend-trading/components/_image-text-switching.tsx:144]: We can also start refactoring these `Show.Mobile` into `Mobile` from `visibility.tsx` and respectively for the `Show.Desktop` as well 🙏

Comment (niloo-fs) [reply src/pages/landing/weekend-trading/components/_image-text-switching.tsx:144]: should I do it in this PR?

**Review by sean-binary — DISMISSED** [review]

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**